### ISSUE TITLE
Transferring to adopted-ember-addons

### ISSUE THREAD

**Issue by RobbieTheWagner** [adopted-ember-addons/ember-inputmask#163]
Hi @brandynbennett, thanks so much for your work on this addon! Would you be open to transferring it to adopted-ember-addons so we can get it updated to latest Ember standards?

**Comment by brandynbennett** [comment]
Yes, I'd be happy to do that. I tried once and for

**Comment by brandynbennett** [comment]
Yes I would be happy to. I tried to transfer ownership once and for some reason it didn't go through. Let me try again

**Comment by brandynbennett** [comment]
It has been transferred
